Part2-API-Schemas/openapi.yaml

```diff
@@ -566,7 +551,7 @@ components:
         aasIds: 
           items: 
             type: string
-            pattern: "^[\\x09\\x0A\\x0D\\x20-\\uD7FF\\uE000-\\uFFFD\\U00010000-\\U0010FFFF]*$"
+            pattern: "^([\\x09\\x0a\\x0d\\x20-\\ud7ff\\ue000-\\ufffd]|\\ud800[\\udc00-\\udfff]|[\\ud801-\\udbfe][\\udc00-\\udfff]|\\udbff[\\udc00-\\udfff])*$"
```

Comment:
Is that really required to allow tab, line feed and carriage return (newline) for an id?

Reply:
It's what the metamodel defines, therefore, we have to use exactly this pattern...

Reply:
We wanted to support IRI, does IRI allow  tab, line feed and carriage return (newline)?